composer.json

```diff
@@ -30,7 +30,7 @@
     "replace": {
         "erusev/parsedown": "1.7.4",
         "erusev/parsedown-extra": "0.8.1",
-        "filp/whoops": "2.14.6",
+        "filp/whoops": "2.14.5",
```

Comment:
Aktuell wird die Fehlerzeile im Codeausschnitt in Whoops nicht markiert.
Auch wird der Ausschnitt komisch gewählt, die Fehlerzeile ist die letzte Zeile, statt mittig im Ausschnitt.
https://github.com/filp/whoops/issues/747

Problem an dem Downgrade ist aber, dass die Whoops-Version ein Security-Update für Prism enthielt:
https://github.com/filp/whoops/pull/741

So wie ich das verstehe, betrifft uns das Security-Problem aber nicht wirklich, aus zwei Gründen:
- Whoops ist sowieso nur für Backend-Admins (und Debug-Mode) verfügbar
- Über Prism wird nur unser Source Code ausgegeben, kein Source Code vom Website-User.

Ich denke also, der Revert ist ok. Aber so ganz wohl ist mir dabei natürlich auch nicht.
Mir ist das Problem schon länger aufgefallen, und hatte auf einen Fix in Prism gehofft und daher gewartet.

Das Problem ist bisher in keinem REDAXO-Release enthalten.

Reply:
:+1: